app/controllers/management_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
   end
 
   def index_to_solr
-    @metadata_source = FixtureIndexingService.index_fixture_data(params[:metadata_source])
+    @ms = FixtureIndexingService.index_fixture_data(params[:ms])
```

Comment:
I don't feel like this is better. are we trying to appease rubocop or something?

Reply:
Agree this is ugly - it was to get it out of the way so that I could turn MetadataSource into its own model, and this was blocking the creation of the routes

I'll see whether, now that MetadataSource is its own model, this can be refactored.